apps/web/components/ShippingMethod/ShippingMethod.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div data-testid="shipping-method" class="md:px-4 my-6">
+    <div class="flex justify-between items-center">
+      <h3 class="text-neutral-900 text-lg font-bold">{{ $t('shippingMethod.heading') }}</h3>
+    </div>
+    <div class="mt-4">
+      <ul
+        v-if="shippingMethods && shippingMethods.methods"
```

Comment:
```suggestion
        v-if="shippingMethods?.methods"
```
Should work as well, you can try :)

Reply:
https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/68/commits/b175a7b562d949619ff958ac0b3c14c489f1d84f it works, thanks :)